src/javascript/app_2/App/Components/Elements/Label/label.jsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React      from 'react';
+import PropTypes  from 'prop-types';
+
+const available_modes = [
```

Comment:
I think this should not be global. Is there any reason why it should be?

Reply:
@ahkuma `Label` component is a function component, so it doesn't have a `static` modifier. That's meant to make it a static variable as we do not want to reinitialise it every time we call the `Label` component.

Reply:
@msamprz but what happens when there are other variables somewhere that uses available_modes? If someone made a mistake it won't be caught and they will be using this instead of what they intended.

Reply:
@msamprz ping